### ISSUE TITLE
[Reproduction Question] AUROC Discrepancy When Reproducing XFMamba Tiny Mode

### ISSUE THREAD

**Issue by TIanCat** [XZheng0427/XFMamba#4]
I’m a researcher currently learning and reproducing your XFMamba work. First, thank you for open-sourcing the code and weights—it’s been incredibly helpful for validating the method!

However, I’m encountering a large gap between my reproduced metrics and the values reported in your paper, and I’d appreciate any insights you might have. Below are the key details:

1. Environment & Setup
Python version: 3.10.18
PyTorch version: 2.1.2+cu118
Dataset: CheXpert , strictly following the paper’s train/val/test split]
Model: XFMamba Tiny

2. Reproduction Steps & Results
I tested two scenarios, both using the same test set:

(1) Use the pre-provided tiny model weights (from the repo) for inference：0.7234 

<img width="509" height="317" alt="Image" src="https://github.com/user-attachments/assets/a44ae872-7ab7-469e-98c2-00e65376eb40" />

(2) Re-train the tiny model strictly following the repo’s config: 0.7858 

<img width="498" height="315" alt="Image" src="https://github.com/user-attachments/assets/cf0c1d6d-c2c3-4a5e-8b06-8aa3fca580db" />

but Paper’s AUROC	:  0.917

**I treated uncertain labels (-1) as positive (1) during training and inference, as this detail was not mentioned in the repo’s code—and I’m unsure if this aligns with your original setup.**

Could you help point out potential reasons for this metric gap?
